### PR TITLE
feat(snackbar): add option to force action btn display on new line

### PR DESCRIPTION
**Pull request by soykje**
**TASK**: #2147 

### Description, Motivation and Context
Following design specs update, we want to improve item layout following way:
Image

For more infos: https://www.figma.com/design/0QchRdipAVuvVoDfTjLrgQ/Component-Specs-of-Spark?node-id=43448-28117&t=7KNS8UY1D2Fu3ifo-4

### Types of changes
- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🧾 Documentation
- [x] 🧪 Test
- [x] 🧠 Refactor
- [x] 💄 Styles

### Screenshots - Animations
image

### CODE REVIEW

File: packages/components/snackbar/src/Snackbar.stories.tsx [hunk 1]
```diff
@@ -217,17 +217,32 @@ export const Action: StoryObj = {
...
+            Display snackbar with action (default)
+          </Button>
+
+          <Button
+            onClick={() =>
+              addSnackbar({
+                message: "You're done! But maybe you should care about what you just did?",
+                actionLabel: 'Undo',
+                onAction: () => console.log('Undone'),
+                forceActionOnNewline: true,
```
Comment (Powerplex) [inline packages/components/snackbar/src/Snackbar.stories.tsx:239]: I would just remove `force` -> `actionOnNewLine`

Comment (soykje) [reply packages/components/snackbar/src/Snackbar.stories.tsx:239]: No strong opinion about this :neutral_face: Wdyt @andresin87 ?

Comment (Powerplex) [reply packages/components/snackbar/src/Snackbar.stories.tsx:239]: Not a big issue if we keep as it is, it's just that "force" make it sound like an "undesired" feature.

Comment (soykje) [reply packages/components/snackbar/src/Snackbar.stories.tsx]: Ok then :+1:

**Review by andresin87 — APPROVED** [review]

**Review by Powerplex — APPROVED** [review]

**Pull request merged**